interview-questions.md

```diff
@@ -5,56 +5,102 @@ Answer the following questions. First, without external resources. Challenge you
 
 2. What are the 5 HTTP verbs?
 
-  Your answer:
+  Your answer: Read, Put, Post, Update, Patch 
+
+I realize I have confused these terms and thought they were interchangeable.
 
   Researched answer:
 
+GET
+The GET method requests a representation of the specified resource. Requests using GET should only retrieve data and should have no other effect. \
 
+HEAD
+The HEAD method asks for a response identical to that of a GET request, but without the response body. This is useful for retrieving meta-information written in response headers, without having to transport the entire content.
 
-3. When creating a basic (stateless) class component in React, what are the necessary elements needed to render "Hello World" on the page?
+POST
+The POST method requests that the server accept the entity enclosed in the request as a new subordinate of the web resource identified by the URI. 
 
-  Your answer:
+PUT
+The PUT method requests that the enclosed entity be stored under the supplied URI. 
 
-  Researched answer:
+DELETE
+The DELETE method deletes the specified resource.
 
+TRACE
+The TRACE method echoes the received request so that a client can see what (if any) changes or additions have been made by intermediate servers.
 
+OPTIONS
+The OPTIONS method returns the HTTP methods that the server supports for the specified URL.
 
-4. What is JSX?
+CONNECT
+The CONNECT method converts the request connection to a transparent TCP/IP tunnel, usually to facilitate SSL-encrypted communication (HTTPS) through an unencrypted HTTP proxy.
+
+PATCH
+The PATCH method applies partial modifications to a resource.
+
+
+3. When creating a basic (stateless) class component in React, what are the necessary elements needed to render "Hello World" on the page?
 
   Your answer:
+You'll need to call the React script from either from the server hosting the React or self hosting it. Then instruct the React component to inherit the characteristics it needs to do be able to do work. The text is placed within a render that is in a master div. 
 
   Researched answer:
+A React stateless components simply returns a value which you input in your view. There are no states but methods may be there. It is very easy to create maintainable and readable code. The stateless component will break the components into small parts and pass data through props. This is also called as Pure Function. There is no ‘this’ keyword there. And it also has no lifecycle method like componentDidMount.
```

Comment:
this question was a little confusing and your answer is great. but just so you know, we were looking for: import, class name extends, render, return, div/React.Fragment would be a bonus, export included in this answer somehow